stable/mlflow/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mlflow.fullname" . }}
+  labels:
+    {{- include "mlflow.labels" . | nindent 4 }}
+  annotations:
```

Comment:
Remove this annotation, its no longer relevant

Reply:
True, good catch! removed